docs/src/api.md

```diff
@@ -25,6 +25,29 @@ Geometry.AbstractPoint
 Geometry.float_type
 ```
 
+Points represent _locations_ in space (not to be confused with _positions_,
+_position vectors_ or _location vectors_ in geometry). Vectors, on the other hand,
```

Comment:
I still don't understand the distinction between a coordinate and a position.

Reply:
This comment was here only for the people with a geometry background (like me) that might be confused because in geometry the terminology _position_ can be used for [_position vector_ or _location vector_](https://en.wikipedia.org/wiki/Position_(geometry)#:~:text=The%20term%20%22position%20vector%22%20is,affine%20spaces%20of%20any%20dimension.), i.e., a Euclidean vector that represents the position of a point P in space in relation to an arbitrary reference origin O. But if not useful for this community, I can certainly remove it.

Reply:
Having thought about this a bit more, I think the more useful distinction is that a "point' describes a position via a particular set of coordinates, but that that the coordinates need not be Cartesian.

Reply:
Yes. So do we prefer to rephrase the sentence above or is it good as is?

Reply:
I would prefer to rephrase it if possible, as I think the current wording is confusing.

Reply:
Sure, what do you have in mind?

Reply:
@simonbyrne , I applied your other suggestions. For this point in particular I am not sure what you had in mind. As I said, we can just remove the part in parentheses. I don't mind.